client/components/UserSettings/AdminSettings.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { LoadingStatus, useTriggerLoad } from '../common/LoadingStatus';
+import { useThemedModal, THEMES } from '../../hooks/useThemedModal';
+import { dates } from 'shared';
+
+const AdminSettings = ({ latestTestPlanVersion, refetch }) => {
+  const { triggerLoad, loadingMessage } = useTriggerLoad();
+  const {
+    themedModal,
+    showThemedModal,
+    setShowThemedModal,
+    setThemedModalTitle,
+    setThemedModalContent,
+    setThemedModalType
+  } = useThemedModal({
+    type: THEMES.SUCCESS,
+    title: 'Success!'
+  });
+
+  const handleImportTests = async () => {
+    await triggerLoad(async () => {
+      try {
+        const response = await fetch('/api/test/import', { method: 'POST' });
+        if (!response.ok) {
+          throw new Error(
+            `Failed to import the latest Test Plan Versions: ${response.status}`
+          );
+        }
+
+        // Success
+        setThemedModalType(THEMES.SUCCESS);
+        setThemedModalTitle('Success!');
+        setThemedModalContent(
+          <>The latest Test Plan Versions have been imported.</>
+        );
+        setShowThemedModal(true);
+        await refetch();
+      } catch (e) {
+        // Failed, show themed message
+        setThemedModalType(THEMES.DANGER);
+        setThemedModalTitle('Error!');
+        setThemedModalContent(<>{e.message}</>);
+        setShowThemedModal(true);
+      }
+    }, 'Importing latest Test Plan Versions');
+  };
+
+  return (
+    <LoadingStatus message={loadingMessage}>
+      <section>
+        <h2>Actions</h2>
+        <p>Use the following actions below to perform a certain function:</p>
```

Comment:
This text feels somewhat superfluous as it is now. Perhaps it could be worthwhile to indicate that these are actions that can be performed by admins only?

Reply:
Should be addressed in https://github.com/w3c/aria-at-app/pull/1290/commits/f5193e99d415fd6276f79e7d40c333522fabcc4f